src/main/java/au/org/emii/ncdfgenerator/CreateWritable.java

```diff
@@ -2,29 +2,44 @@
 package au.org.emii.ncdfgenerator;
```

Comment:
Cool.  There's also https://github.com/aodn/netcdf_filters1 which I guess can be deleted.